CompetitionTemplate/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,60 @@
+package competition.subsystems.elevator;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+
+
+@Singleton
+public class ElevatorSubsystem extends BaseSubsystem {
+	
+	@Inject
+	public ElevatorSubsystem() {
+		// TODO Auto-generated constructor stub
+	
+	}
+
+	/**
+	 * the elevator will rise by certain speed, the speed may change
```

Comment:
Most functions should be described in a verb-y way:
"Raises the elevator. Power is controlled by a property."

Reply:
Got it!